src/plugins/recordTypes/collectionobject/fields.js

```diff
@@ -1384,6 +1384,67 @@ export default (configContext) => {
             },
           },
         },
+        objectClassList: {
```

Comment:
Make this `objectClassGroupList` to be consistent with naming of other repeating groups.